test/units/testsuite-74.firstboot.sh

```diff
@@ -119,6 +119,15 @@ grep -q "^root:x:0:0:.*:/bin/barshell$" "$ROOT/etc/passwd"
 grep -q "^root:$ROOT_HASHED_PASSWORD2:" "$ROOT/etc/shadow"
 grep -q "hello.world=0" "$ROOT/etc/kernel/cmdline"
 
+# Test that --reset removes all files configured by firstboot.
+systemd-firstboot --root="$ROOT" --reset
+[[ ! -e /etc/locale.conf ]]
```

Comment:
I guess you meant

```suggestion
[[ ! -e "$ROOT/etc/locale.conf" ]]
```
? Ditto for the lines below.

Reply:
Oops